number_parser/parser.py

```diff
@@ -267,6 +267,35 @@ def parse_number(input_string, language=None):
         return int(number_built[0])
     return None
 
+def parse_fraction(input_string, language=None):
+    """Converts a single number written in fraction to a numeric type"""
+    if not input_string.strip():
+        return None
+
+    if language is None:
+        language = _valid_tokens_by_language(input_string)
+
+    fraction_separators = ["divided by", "over", "by", "/"]
```

Comment:
```suggestion
    FRACTION_SEPARATORS = ["divided by", "over", "by", "/"]
```

I prefer to use capital letters here. Ideally, in the future this will come from the lang_data (`en.json`), we can work on this on a new, separated, PR.

Reply:
Sure, It also has to incorporate the fraction separators in other languages